lib/purchases_flutter.dart

```diff
@@ -45,6 +50,8 @@ class Purchases {
 
   static CustomerInfo? _lastReceivedCustomerInfo;
 
+  static LogHandler? _logHandler;
```

Comment:
Hmm we currently don't log much/anything in the hybrids. If we do in the future, we should probably redirect those logs over here as well... Nothing to do here right now, just something to think about.